dev/io.openliberty.jaxws.security_fat/fat/src/com/ibm/ws/jaxws/security/fat/TransportSecurityUsingDispatchTest.java

```diff
@@ -28,135 +32,70 @@
 @Mode(Mode.TestMode.FULL)
 public class TransportSecurityUsingDispatchTest extends AbstractJaxWsTransportSecurityTest {
```

Comment:
Can you do the same thing with this class? I don't think there is a reason to have this as it's own separate test class, when essentially it's doing the same tests as the BasicAuthWithoutSSL and BasicAuthWithSSL. The tests that apply to the BasicAuthWithoutSSL can be moved to that class, and the ones that apply to the BasicAuthWithSSL can move to that class. And then we can just remove this test class entirely.